### ISSUE TITLE
Generate and post report upon TLS-pinning failure

### ISSUE THREAD

**Issue by calexandru2018** [ProtonMail/proton-python-client#19]
A report should be generated and sent in case of TLS-pinning failure, possibly containing the following info:
- appVersion
- appPlatform
- appPlatformVersion
- serverHostname
- serverPort
- dateTime
- notedHostname
- includeSubdomains
- knownPins
- validationResult
- knownPinsExpirationDate

**Comment by Alex1Jin** [comment]
would you please tell me what api_url is ? thx a lot

**Comment by butla** [comment]
Yup, please document the API endpoint. Been searching for it for a bit and can't find it.

Is anyone in the company even looking at this repository? If this project is abandoned it would be good to at least document that in the README.

EDIT:
Asked ChatGPT for it (it gave the correct answer :) ), but then I found it already mentioned in this issue: https://github.com/ProtonMail/proton-python-client/issues/34
It's https://api.protonmail.ch
